documentation/2.0/content/userguide/target_env.md

```diff
@@ -37,31 +37,45 @@ If a variable file is specified on the tool's command line using the `-variable_
 
 These target environment configurations are included in the WebLogic Deploy Tooling installation.
 
-#### The WebLogic Kubernetes Operator target
+#### The WebLogic Kubernetes Operator targets
 
-You can apply this target environment by providing the command-line argument `-target wko`. It will provide this additional processing:
+You can use these targets to customize the model and create a domain resource file for use with WebLogic Kubernetes Operator. There are three targets for specific domain home source types:
```

Comment:
Do you want to make [domain home source types](https://oracle.github.io/weblogic-kubernetes-operator/userguide/managing-domains/choosing-a-model/) a cross-reference, because it is an unusual term that has probably not yet been defined in the WDT docs (though perhaps, Model in Image may have been).

Reply:
Added this